airbyte-integrations/connectors/source-ashby/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,13 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "users",
```

Comment:
you must add all streams to run the integration tests.

Reply:
✅